scope_ascom.cpp

```diff
@@ -179,6 +184,7 @@ bool ScopeASCOM::HasSetupDialog() const
 void ScopeASCOM::SetupDialog()
 {
     DispatchObj scope;
+    bool prevRegistered = AlreadyPresentInGit(m_gitEntry);

```

Comment:
let's reverse these two lines to keep the scope of of the variable `scope` to the minimum required (and for for clarity -- it is tightly associated the the Create() call)
```suggestion
    bool prevRegistered = m_gitEntry.IsRegistered();
    DispatchObj scope;
```